src/Bicep.Core/Features/FeatureProvider.cs

```diff
@@ -55,6 +54,8 @@ public FeatureProvider(RootConfiguration configuration, IFileExplorer fileExplor
 
         public bool ExtensibilityV2EmittingEnabled => ReadBooleanEnvVar("BICEP_EXTENSIBILITY_V2_EMITTING_ENABLED", defaultValue: false);
 
+        public bool ModuleExtensionConfigsEnabled => configuration.ExperimentalFeaturesEnabled.ModuleExtensionConfigs;
```

Comment:
I noticed the emission of v2 format is read by env var, but I think if the module extension configs feature is enabled, it should automatically enable this. Thoughts?

Reply:
Yeah I think so.